src/cf-atomic-component/src/components/AtomicComponent.js

```diff
@@ -64,7 +62,6 @@ assign( AtomicComponent.prototype, Events, classList, {
       if ( classList.contains( this.element, modifier.ui.base ) ) {
         if ( modifier.initialize ) {
           this.initializers.push( modifier.initialize );
-          delete modifier.initialize;
```

Comment:
Something in the initialization of AtomicComponent makes the state leak. Without removing this line the tests will not pass, because they will share the state of `modifier.initialize` being deleted. To test, ensure this line is present in the AtomicComponent used by cf-tables and see that the tests fail. Then `xit` all but the last test in `Table-spec.js` and see that they pass. Enable the first test (so only the first and last are running) and see that they fail again. Somehow the state is carried between tests by `Table.init()`, which is bad news. I believe we can delete this line as the tests still pass without it, but I'm not 100% certain as this appears to be a means to indicate initialization has already happened on the modifiers (instead of say, placing a flag in the DOM). @mistergone Any insight on this?